src/navigation-bar.tsx

```diff
@@ -129,11 +242,1008 @@ type MainNavigationBarPropsBase = {
+                })}
+                onMouseEnter={() => openCurrentSectionMenu(true)}
+                onMouseLeave={() => cancelDebouncedOpenSectionMenu(index)}
+                onFocus={() => {
+                    if (menu && !hasCustomInteraction) setIsMenuControlFocused(true);
+                }}
+                onBlur={() => {
+                    if (menu && !hasCustomInteraction) setIsMenuControlFocused(false);
+                }}
+            >
+                <BaseTouchable
+                    {...getSectionInteractiveProps(touchableProps as InteractiveProps)}
+                    aria-label={
+                        !hasCustomInteraction
+                            ? `${section.title}, ${texts.mainNavigationBarOpenSectionMenu || t(tokens.mainNavigationBarOpenSectionMenu)}`
+                            : undefined
+                    }
+                    aria-haspopup={!hasCustomInteraction}
+                    aria-expanded={!hasCustomInteraction ? isSectionMenuOpen : undefined}
+                    className={classnames(
+                        styles.section,
+                        {
+                            [styles.selectedSectionVariantes[variant === 'inverse' ? 'inverse' : 'default']]:
+                                index === selectedIndex,
+                        },
+                        styles.textWrapperVariants[variant === 'inverse' ? 'inverse' : 'default']
+                    )}
+                >
+                    <Text3 regular color="inherit">
+                        {title}
+                    </Text3>
+                </BaseTouchable>
+            </div>
+            {menu && (
+                <>
+                    {hasCustomInteraction && (
+                        <div
+                            className={styles.desktopMenuSectionArrowContainer}
+                            onFocus={() => setIsMenuControlFocused(true)}
+                            onBlur={() => setIsMenuControlFocused(false)}
+                        >
+                            <BaseTouchable
+                                className={styles.desktopMenuSectionArrow}
+                                aria-label={`${section.title}, ${texts.mainNavigationBarOpenSectionMenu || t(tokens.mainNavigationBarOpenSectionMenu)}`}
+                                aria-haspopup
+                                aria-expanded={isSectionMenuOpen}
+                                onPress={() => {
+                                    if (isMenuControlFocused) {
+                                        menuButtonOnPress();
+                                    }
+                                }}
+                                style={{
+                                    pointerEvents: isMenuControlFocused ? 'auto' : 'none',
+                                    opacity: isMenuControlFocused ? 1 : 0,
+                                }}
+                            >
+                                <IconChevronLeftRegular
```

Comment:
missing the chevron when you have sections without onpress
https://tinyurl.com/23exq4zb

Reply:
After internal discussions, we have decided not to include these details for the following reasons:

1. Including this detail would require us to cancel certain accessibility events, which could negatively impact overall accessibility.
2. The focus ring would not display the chevron correctly, and we would need to adjust the button width to accommodate it.
3. These details are intended to enhance consistency when mixing multi-interactive tabs (menu + navigation) with interactive tabs (navigation), but this is considered a corner case.